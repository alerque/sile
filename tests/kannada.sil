\begin[papersize=a6]{document}
\nofolios
\noindent
\font[family=Libertinus Serif,size=16pt]
Hello SILE! \font[family=Noto Sans Kannada,language=kn]{ನಮಸ್ಕಾರ :)}
\end{document}
