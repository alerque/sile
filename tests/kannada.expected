Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	33.9139
Set font 	Libertinus Serif;16;400;;normal;;LTR
T	41 70 77 77 80	(Hello)
Mx 	55.0481
T	52 42 45 38	(SILE)
Mx 	84.9201
T	2	(!)
Mx 	94.3502
Set font 	Noto Sans Kannada;16;400;;normal;;LTR
T	39 44 163 57 97 46	(ನಮಸ್ಕಾರ)
Mx 	161.5164
T	422	(:)
Mx 	165.8054
T	405	())
End page
Finish
